extension-manifest-v3/src/store/options.js

```diff
@@ -29,6 +29,7 @@ export const SYNC_OPTIONS = [
   'trackerCount',
   'wtmSerpReport',
   'serpTrackingPrevention',
+  'experimentalCosmeticFilters',
```

Comment:
I would name this option just `experimental` - as this not mean only cosmetics or filters (at all for DNR version)